RELEASE-NOTES.md

```diff
@@ -1,21 +1,31 @@
 # Release Notes
 
-## PyMC3 vNext (3.11.1)
-
+## PyMC3 vNext (TBD)
 ### Breaking Changes
++ ...
+
+### New Features
++ ...
+
+### Maintenance
++ ...
+
+## PyMC3 3.11.1 (12 February 2021)
 
 ### New Features
 - Automatic imputations now also work with `ndarray` data, not just `pd.Series` or `pd.DataFrame` (see[#4439](https://github.com/pymc-devs/pymc3/pull/4439)).
 - `pymc3.sampling_jax.sample_numpyro_nuts` now returns samples from transformed random variables, rather than from the unconstrained representation (see [#4427](https://github.com/pymc-devs/pymc3/pull/4427)).
 
 ### Maintenance
 - We upgraded to `Theano-PyMC v1.1.2` which [includes bugfixes](https://github.com/pymc-devs/aesara/compare/rel-1.1.0...rel-1.1.2) for warning floods and compiledir locking (see [#4444](https://github.com/pymc-devs/pymc3/pull/4444))
-- `Theano-PyMC v1.1.2` also fixed an important issue in `tt.switch` that affected the behavior of several PyMC distributions, including at least the `Bernoulli` and `TruncatedNormal` (see[#4448](https://github.com/pymc-devs/pymc3/pull/4448))
+- `Theano-PyMC v1.1.2` also fixed an important issue in `tt.switch` that affected the behavior of several PyMC distributions, including at least the `Bernoulli` and `TruncatedNormal` (see [#4448](https://github.com/pymc-devs/pymc3/pull/4448))
```

Comment:
This sounds quite disconcerting without more context. This only affected the bounds, not the inference, right?

Reply:
I.e. the question if people got wrong results.

Reply:
It affected inference as well:
1. For the Bernoulli only on the edge case where all observed values were 0 or 1
2. For the TruncatedNormal more often, depending on whether mu / sigma were being broadcasted (gradient failed, but not logp), but not otherwise (as seen in the new pymc-examples Truncated Regression).

Reply:
In that case we should make a bigger deal out of this. Maybe we should write a wiki page describing this in more detail and link it here. But if people potentially got wrong answers and need to redo analyses we owe it to our users to be fully transparent about this and warn them properly.

Reply:
I can do it on Monday. If someone wants to write up in the meanwhile, here is the illustrating example for the TruncatedNormal:

```python
import arviz as az
import numpy as np
import pymc3 as pm


np.random.seed(2021)
x = np.random.normal(0, 2, size=5000)
obs = x[(x >= -1) & (x <= 2)]

# Problems with NUTS (but not Metropolis, since it is a gradient bug) when mu has to be broadcasted
with pm.Model() as m1:
    mu = pm.Normal('mu', 0, 5)
    x = pm.TruncatedNormal('x', mu=mu, sigma=2, lower=-1, upper=2, observed=obs, shape=len(obs))
    trace1 = pm.sample()

print(az.summary(trace1))  # convergence stats look bad

# Fine when mu does not need to be broadcasted
with pm.Model() as m2:
    mu = pm.Normal('mu', 0, 5)
    x = pm.TruncatedNormal('x', mu=mu+np.zeros_like(obs), sigma=2, lower=-1, upper=2, observed=obs, shape=len(obs))
    trace2 = pm.sample()

print(az.summary(trace2))  # converge stats look fine
```

Reply:
Agree. @ricardoV94 can you give me a more precise description of the issue to replace the current one in the release notes?

We can also link to an issue/PR/wiki page even if the writeup is not done yet.
Let's get the fix released!

Reply:
`Theano-PyMC v1.1.2` also fixed an important issue in `tt.switch` that affected the behavior of several PyMC distributions, including at least the following cases:
1.  `Bernoulli` when all the observed values were the same (e.g., `[0, 0, 0, 0, 0]`)
2.  `TruncatedNormal` when `sigma` was constant and `mu` was being automatically broadcasted to match the shape of observations.

(see [#4448](https://github.com/pymc-devs/pymc3/pull/4448)).